src/energyplus/Test/ZoneHVACTerminalUnitVariableRefrigerantFlow_GTest.cpp

```diff
@@ -291,3 +364,186 @@ TEST_F(EnergyPlusFixture, ForwardTranslatorZoneHVACTerminalUnitVariableRefrigera
+    }
+  }
+
+  // 2: The VRF TU
+  {
+    WorkspaceExtensibleGroup w_eg2 = idf_branch.extensibleGroups()[1].cast<WorkspaceExtensibleGroup>();
+
+    EXPECT_EQ("ZoneHVAC:TerminalUnit:VariableRefrigerantFlow", w_eg2.getString(BranchExtensibleFields::ComponentObjectType).get());
+    EXPECT_EQ(w_eg2.getString(BranchExtensibleFields::ComponentName).get(), vrf.name());
+
+    EXPECT_EQ(w_eg2.getString(BranchExtensibleFields::ComponentInletNodeName).get(), vrf.inletNode().get().nameString());
+
+    EXPECT_EQ(w_eg2.getString(BranchExtensibleFields::ComponentOutletNodeName).get(), vrf.outletNode().get().nameString());
+
+    auto idf_vrf_ = w_eg2.getTarget(BranchExtensibleFields::ComponentName);
+    ASSERT_TRUE(idf_vrf_);
+    EXPECT_EQ(vrf.nameString(), idf_vrf_->nameString());
+    // No OutsideAirMixer, since it's on a AirLoopHVAC
+    EXPECT_FALSE(idf_vrf_->getTarget(ZoneHVAC_TerminalUnit_VariableRefrigerantFlowFields::OutsideAirMixerObjectName));
+    // Fan is translated
+    EXPECT_TRUE(idf_vrf_->getTarget(ZoneHVAC_TerminalUnit_VariableRefrigerantFlowFields::SupplyAirFanObjectName));
+  }
+
+  // Let's see if we got the SPMs
+  WorkspaceObjectVector spm_scheduleds(w.getObjectsByType(IddObjectType::SetpointManager_Scheduled));
+  // There's the one I created for the supply outlet node
+  ASSERT_EQ(1u, spm_scheduleds.size());
+  {
+    auto thisSPM = spm_scheduleds[0];
+    EXPECT_EQ(spm.nameString(), thisSPM.nameString());
+    EXPECT_EQ("Temperature", thisSPM.getString(SetpointManager_ScheduledFields::ControlVariable).get());
+    EXPECT_EQ(sch.nameString(), thisSPM.getString(SetpointManager_ScheduledFields::ScheduleName).get());
+    EXPECT_EQ("Supply Outlet Node", thisSPM.getString(SetpointManager_ScheduledFields::SetpointNodeorNodeListName).get());
+  }
+
+  WorkspaceObjectVector spm_mixedairs(w.getObjectsByType(IddObjectType::SetpointManager_MixedAir));
+  EXPECT_EQ(2u, spm_mixedairs.size());
+  for (const auto& spm_mixedair : spm_mixedairs) {
+    EXPECT_EQ("Temperature", spm_mixedair.getString(SetpointManager_MixedAirFields::ControlVariable).get());
+    EXPECT_EQ("Supply Outlet Node", spm_mixedair.getString(SetpointManager_MixedAirFields::ReferenceSetpointNodeName).get());
+    EXPECT_EQ(vrf.nameString() + " Heating Coil Outlet Node", spm_mixedair.getString(SetpointManager_MixedAirFields::FanInletNodeName).get());
+    EXPECT_EQ("Supply Outlet Node", spm_mixedair.getString(SetpointManager_MixedAirFields::FanOutletNodeName).get());
+    EXPECT_TRUE(spm_mixedair.nameString().find("OS Default SPM") != std::string::npos);
+    if (spm_mixedair.nameString().find("Mixed Air") != std::string::npos) {
+      EXPECT_EQ("Mixed Air Node", spm_mixedair.getString(SetpointManager_MixedAirFields::SetpointNodeorNodeListName).get());
+    } else {
+      EXPECT_EQ("VRF TU on Outdoor Air Outlet Node", spm_mixedair.getString(SetpointManager_MixedAirFields::SetpointNodeorNodeListName).get());
+    }
+  }
```

Comment:
Check that SPM:MixedAir correctly picked up the fan that is inside the VRF TU on the main branch